### ISSUE TITLE
won't scrobble after chrome update

### ISSUE THREAD

**Issue by chath-tsu** [web-scrobbler/web-scrobbler#3167]
it used to scrobble just fine however it stoped working completely after updating chrome. it says "this website is not supported" on all websites

extension version 2.55.0
chrome version 99.0.4844.51
windows 10.0.19043 N/A Build 19043

tried restarting chrome and system, tried re-login lastfm in the extension, and tried removing the extension and reinstalling from chrome store, still doesn't work

**Comment by Mavoy** [comment]
Yes, it's the same for me. Unfortunately, we need to use the other browser temporarily. Hopin' for quick fix!

**Comment by inverse** [comment]
Could you try following the steps in https://github.com/web-scrobbler/web-scrobbler/issues/3166#issuecomment-1065272378

**Comment by chath-tsu** [comment]
> Could you try following the steps in [#3166 (comment)](https://github.com/web-scrobbler/web-scrobbler/issues/3166#issuecomment-1065272378)

i tried that and still doesn't work :'(

**Comment by Mavoy** [comment]
I completely forgot to check the other topic, thank you ;) It worked for me

**Comment by inverse** [comment]
@chath-tsu are you still affected?

**Comment by chath-tsu** [comment]
> @chath-tsu are you still affected?

it's now fixed for me! thank you
